detekt_custom.yml

```diff
@@ -503,8 +503,11 @@ datadog:
       - "androidx.compose.ui.graphics.Color(kotlin.Long)"
       - "androidx.compose.ui.graphics.Color.toArgb()"
       - "androidx.compose.ui.layout.LayoutCoordinates.positionInWindow()"
+      - "androidx.compose.ui.layout.LayoutInfo.getModifierInfo()"
       - "androidx.compose.ui.unit.Density(kotlin.Float, kotlin.Float)"
       - "androidx.compose.ui.geometry.Size.copy(kotlin.Float, kotlin.Float)"
+      - "androidx.compose.ui.text.AnnotatedString.getStringAnnotations(kotlin.Int, kotlin.Int)"
+      - "androidx.compose.ui.semantics.SemanticsConfiguration.getOrNull(androidx.compose.ui.semantics.SemanticsPropertyKey)"
```

Comment:
prefer to have it sorted (although it is not sorted right now)

Reply:
updated.